addons/medical_ai/functions/fnc_healingLogic.sqf

```diff
@@ -54,9 +54,19 @@ switch (true) do {
         _treatmentTime = 5;
         _treatmentArgs = [_target, _selection, "FieldDressing"];
     };
+    case (IN_CRDC_ARRST(_target)): {
+        _treatmentEvent = QEGVAR(medical_treatment,cprLocal);
```

Comment:
What if cpr success chance is set to 0? Then they're just extending the cardiac arrest timer but not actually treating the patient (in such a case the only treatment is a PAK I believe).

Reply:
good point, but I believe the PAK needs requires isInStableCondition, so they need to be awake and have a heart rate to use it.

Reply:
I think that requirement might have been removed, but so much has changed I need to double check that 😆

In any case, we should add something to handle this case

Reply:
Would it be more prudent to just give them something along the line of:
```
if(GVAR(cprSuccessChance) == 0) exitWith {
_treatmentEvent = QEGVAR(medical_treatment,cprSuccess);
};
```
as a PAK might be a bit overkill on the AI healing system.

Reply:
Updated to cover this case. I don't know if it was the right way to go about it but it definitely seems to work.